### PR TITLE
Get json request test to pass with other credentials

### PR DESCRIPTION
**Pull request by davidhemphill**
- Rely on system certs
- Make expectation match when other credentials are provided

### CODE REVIEW

File: src/Request/BaseRequest.php [hunk 1]
```diff
@@ -103,7 +103,7 @@ public function sendRequest()
     protected function requestOptions()
     {
         $opts = [
-            'verify' => __DIR__ . '/../../resources/cert.pem',
+            'verify' => true,
```
Comment (mglaman) [inline src/Request/BaseRequest.php:106]: Maybe this should be a config setting passed to the SDK? "Use this cert path instead."

**Comment by mglaman** [comment]
Failure looked like general timeouts / tests running too fast, maybe. Restarted one environment.

File: src/Request/BaseRequest.php (hunk 1 above)
Comment (davidhemphill) [reply src/Request/BaseRequest.php:106]: Maybe so! I'd be fine with using the cert provided in the package, but it just wasn't working in my environment.

**Comment by mglaman** [comment]
@davidhemphill I just merged the bank account PR. I can update the SDK to turn off local certificates via config and fixup JSON tests to ue constants.

**Comment by davidhemphill** [comment]
@mglaman Sounds good.

**Comment by mglaman** [comment]
@davidhemphill see @ccfcd1e2d9370259d6769a5d7b39782cdf5dc1ef

Edit your phpunit.xml and change `TESTS_CERTIFICATE_VERIFY` to `true`. This way tests will run off system and not PEM file.

Reopen new issue if still being wonkers for you.

**Pull request closed without merging**